krun/results.py

```diff
@@ -18,8 +18,8 @@ def __init__(self, config, platform, results_file=None):
 
         # Maps key to results:
         # "bmark:vm:variant" -> [[e0i0, e0i1, ...], [e1i0, e1i1, ...], ...]
-        self.data = dict()      # wall-clock times
-        self.tsr_data = dict()  # timestamp register times
+        self.data = dict()              # wall-clock times
+        self.core_cycles_data = dict()  # core cycles all cores
```

Comment:
I think we discussed this in the past and decided that it would be beneficial to use a normal dict because it ensures that use of a malformed key make an error.